packages/websocket/package.json

```
@@ -1,10 +1,30 @@
 {
   "name": "@solid-primitives/websocket",
-  "version": "0.3.7",
-  "description": "Primitive to create a worker helper",
+  "version": "0.4.0",
+  "description": "Primitive to create a web socket connection",
   "author": "David Di Biase <dave.dibiase@gmail.com>",
+  "contributors": [
+    "Alex Lohr <alex.lohr@logmein.com>"
+  ],
   "license": "MIT",
   "homepage": "https://github.com/solidjs-community/solid-primitives/tree/main/packages/websocket",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/solidjs-community/solid-primitives.git"
+  },
+  "primitive": {
+    "name": "websocket",
+    "stage": 0,
+    "list": [
+      "makeWS",
+      "createWS",
+      "createMessageWS",
+      "makeReconnectingWS",
+      "createReconnectingWS",
+      "makeHeartbeatWS"
```

```suggestion
      "createWS",
      "createMessageWS",
      "createReconnectingWS",
      "makeHeartbeatWS"
```
Not sure if there is a point in listing the non-reactive variants here.
I see them more as additions.
Like you focus on `createWS` as the default, and say btw there is also a variant that doesn't autocleanup.

---

I'm considering marking those primitives that work with other frameworks with solid in order to ease the migration of applications towards solid.